xcom2-launcher/xcom2-launcher/Forms/SettingsDialog.Designer.cs

```diff
@@ -232,9 +225,23 @@ private void InitializeComponent()
 			this.tableLayoutPanel3.RowStyles.Add(new System.Windows.Forms.RowStyle());
 			this.tableLayoutPanel3.RowStyles.Add(new System.Windows.Forms.RowStyle());
 			this.tableLayoutPanel3.RowStyles.Add(new System.Windows.Forms.RowStyle());
+			this.tableLayoutPanel3.RowStyles.Add(new System.Windows.Forms.RowStyle(System.Windows.Forms.SizeType.Absolute, 20F));
 			this.tableLayoutPanel3.Size = new System.Drawing.Size(641, 79);
 			this.tableLayoutPanel3.TabIndex = 6;
 			// 
+			// useSentry
+			// 
+			this.useSentry.AutoSize = true;
+			this.useSentry.Location = new System.Drawing.Point(429, 29);
+			this.useSentry.Name = "useSentry";
+			this.useSentry.Padding = new System.Windows.Forms.Padding(3, 3, 0, 0);
+			this.useSentry.Size = new System.Drawing.Size(170, 20);
+			this.useSentry.TabIndex = 17;
+			this.useSentry.Text = "Send anonymous error reports";
+			this.toolTip.SetToolTip(this.useSentry, "If enabled, unhandled exceptions or other \r\npotential issues are automatically re" +
+        "ported to\r\nour X2CommunityCore Sentry.io account.");
```

Comment:
"unhandled exceptions" is a bit too technical IMO, I think this phrase needs to be simpler to understand for common players

Reply:
Yeah, "unexpected errors" would probably be better.